File: Python/lab15/Service.py
class Service:
    def create_order(self, name):
        print(f"Order: {name}, create")

    def delete_order(self, name):
        print(f"Order: {name}, delete")

    def activate_subscription(self, login):
        print(f"Subscription for user: {login}, activated")

    def call_to_client(self, phone):
        print(f"Calling to: {phone}")

File: Java/lab15/src/main/java/org/example/Service.java
package org.example;

public class Service {
    public void createOrder(String name){
        System.out.println("Order: "+name+", create");
    }
    public void deleteOrder(String name){
        System.out.println("Order: "+name+", delete");
    }
    public void activateSubscription(String login){
        System.out.println("Subscription for user: "+login+", activated");
    }
    public void callToClient(String phone){
        System.out.println("Calling to: "+phone);
    }
}
